app/repositories/raw_data_record_repository.rb

```diff
@@ -92,4 +160,16 @@ def raw_record_date(entity, raw_record)
       entity_date
     end
   end
+
+  def map_psc_es_record(record)
+    RegisterSourcesPsc::CompanyRecord.new(record)
```

Comment:
Is it correct that this is `record`, not `record['data']` or similar? It's different to SK and DK…

Reply:
Discussed.